packages/eslint-plugin/configs/react.js

```diff
@@ -10,7 +10,12 @@ module.exports = {
 			version: 'detect',
 		},
 	},
-	plugins: [ '@wordpress', 'react', 'react-hooks' ],
+	plugins: [
+		'@wordpress',
+		'eslint-plugin-react-compiler',
```

Comment:
```suggestion
		'react-compiler',
```

Reply:
Thanks! Both should work, but the shorter one wins.